src/lib.rs

```diff
@@ -159,6 +159,19 @@ pub trait Reader: Sized {
     fn metadata(&self) -> &Metadata;
     /// Read worksheet data in corresponding worksheet path
     fn worksheet_range(&mut self, name: &str) -> Option<Result<Range<DataType>, Self::Error>>;
+
+    /// Fetch all worksheets
+    fn worksheets(&mut self) -> Vec<(String, Range<DataType>)> {
```

Comment:
I have opted to use a `Vec` over a range, as there is no real reason to request a sheet by name, `worksheet_range` exists for this reason.